### ISSUE TITLE
Tab Split View Keyboard Shortcuts seem faulty?

### ISSUE THREAD

**Issue by guigirl42** [zen-browser/desktop#341]
### What happened?

Invoking the `Tab Split View` via RMB context-menu [as described in https://github.com/zen-browser/desktop/issues/47#issuecomment-2226346344] works well for me. This holds true for two to six tabs [i've not tested more; it might also work well then]. 

Unfortunately though, trying to do it instead via the Keyboard Shortcuts _mostly_ fails.
- two tabs: Ctrl+Alt+V works correctly  :heavy_check_mark: 
- two tabs: Ctrl+Alt+H generates _Vertical_, when it's supposed to be _Horizontal_   ❌
- three+ tabs: Ctrl+Alt+G only generates Vertical, with only two tabs; the others are ignored  ❌

Though i like using the RMB context-menu method, there are times when being able to use kb shortcuts would be helpful, pls. 

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.23

### Severity impact

Medium

### What platform are you seeing the problem on?

Linux

**Comment by mauro-balades** [comment]
Fixed, thanks!

**Comment by guigirl42** [comment]
You're just amazing! Thx :hugs:  

I assume it'll come with 24 or 25? :woman_shrugging:

**Comment by mauro-balades** [comment]
24!

**Comment by guigirl42** [comment]
Champion! :trophy:

**Comment by guigirl42** [comment]
> Fixed, thanks!

Hello again. I've now retested it in 24 a couple of hours ago, & just now in 26, but i'm sorry to advise it seems identically broken to how i previously described. 

It makes me now wonder if maybe all this time i might have been completely misunderstanding HOW to use the keyboard shortcuts... although i always assumed it was simple.  Pls can you advise me, is it now working ok for you at your end?

**Comment by moderation** [comment]
Like @guigirl42 I'm seeing the request for a horizontal split generate a vertical split. Seems like a regression in 26

**Comment by UWUplus** [comment]
Can confirm, neither 26 nor 27 have the horizontal split working, only generating vertical split.

**Comment by guigirl42** [comment]
Sadly, still broken in 28.  Fwiw, i do my tests also in a virgin profile, not only my daily profile, to ensure objectivity.

**Comment by mauro-balades** [comment]
This is fixed, the grid shortcut only will take the next tab. and horizontal also creates a hortizontal view now

**Comment by guigirl42** [comment]
>the grid shortcut only will take the next tab

Sorry, but do you mean that if i wish to "grid" more than 2 tabs, i cannot use the kb shortcut, & instead must only use the RMB context-menu method?

**Comment by mauro-balades** [comment]
I changed, my mind, if multiple tabs are selected, they will be used for split. But if not, only the next available tab.

**Comment by guigirl42** [comment]
OK, i _think_ that i understand, but look forward with interest to the next version so i can test it myself & _ensure_ i have properly understood your intention.

**Comment by guigirl42** [comment]
I've just tested `1.0.0-a.29`. There is sad news, & great news! 

**Sad**:
The kb shortcuts for two tabs vertical & horizontal  _do_ invoke the split view interface, but _only one_ tab is shown. :exploding_head: 

**Great**:
The kb shortcut for grid split works beautifully, i'm so happy! :tada: :partying_face: :woman_dancing: 
I tested it with 4 & 6 tabs, & it reliably worked each time. Given the latest weirdness with H & V as above, i then also tested it just for 2 tabs, & it does work reliably for these, creating H side-by-side tabs, which if the user wants can then be converted to V via the UI icon in the url bar. 

Thank you Dev! :hugs:
